pkg/apis/pipeline/v1/task_types_test.go

```diff
@@ -0,0 +1,68 @@
+/*
+Copyright 2019 The Tekton Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1_test
+
+import (
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	v1 "github.com/tektoncd/pipeline/pkg/apis/pipeline/v1"
+	"github.com/tektoncd/pipeline/test/diff"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+func TestTaskSpec_FindWorkspaceSubstitutionLocationsInTask(t *testing.T) {
+	tests := []struct {
+		name string
+		ts   *v1.TaskSpec
+		want sets.String
+	}{{
+		name: "completespec",
+		ts: &v1.TaskSpec{
```

Comment:
should this handle step template as well?

Reply:
Yes. Will fix. Thanks!